config/govuk_index/migrated_formats.yaml

```diff
@@ -342,7 +341,6 @@ non_indexable:
   - "/googlef35857dca8b812e7.html"
   - "/government"
   - "/government/get-involved"
```

Comment:
🤔 Would I expect to see the "[get involved](https://www.gov.uk/government/get-involved)" page from search such as this? (this is coming from the homepage)

![image](https://user-images.githubusercontent.com/71266765/123656119-8a447b00-d827-11eb-984e-704bed33bd88.png)

Reply:
Confirmed that this appears when searched with quotes.